recipes/renode-cli/recipe.yaml

```diff
@@ -0,0 +1,83 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+  python_min: "3.6"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+    patches:
+      - patches/update-IronPython-Renode_NET.csproj.patch
+      - patches/update-cores.template_NET.csproj.patch
+      - patches/update-IronPython-Unittest_NET.csproj.patch
+      - if: unix
+        then:
+          - patches/resolve-posix-unix.patch
+          - patches/update-unix-informational-version.patch
+        else:
+          - patches/fix-missing-env-common_copy_licenses.sh.patch
+          - patches/update-win-informational-version.patch
+
+build:
+  number: 0
+  skip:
+    - win
```

Comment:
I assume this is disabled for testing?

Reply:
What do you mean? The win build fails as something from the build_env cannot be deleted, so I skip it for the staged-recipe, I was planning to solve and add it to the feedstock along with the other arch

Reply:
I didn't see your other comment about disabling Windows build earlier.  For now, maybe we can leave it enabled so we can make sure things are still working after the cleanup.  And maybe someone will be able to figure why the build_env is clean up is failing as well.